Fix: PDF pages (update metadata code snippet) (#32)
aspose-pdf-cloud/Aspose.PDF-Products: pdf/nodejs/metadata/remove/_index.md

{{% /blocks/products/pf/agp/text %}}

1. Create an object to connect to the Pdf.Cloud API
1. Upload your document file

> 1. Create a new XmpMetadataProperty with Your key and null as value for delete XmpMetadataProperty

Reviewer: ```suggestion
1. Create a new XmpMetadataProperty.  Set the name of the property you want to delete as the Key and `null` as the value
```

> 1. Delete XMpMetadataProperty in the document using postXmpMetadata() function

Reviewer: ```suggestion
1. Delete Metadata in the document using postXmpMetadata() function
```

            const filePath = path.join(configParams.LOCAL_FOLDER, configParams.LOCAL_RESULT_DOCUMENT_NAME);
            await fs.writeFile(filePath, changedPdfData.body);
            console.log("Downloaded: " + filePath);
        },

        async getMetadata () {
            const responseMetadata = await pdfApi.getXmpMetadataJson(configParams.PDF_DOCUMENT_NAME);

            if (responseMetadata.response.status == 200)
            {
                const props = responseMetadata.body.properties;
                props.forEach((prop) =>{
                    console.log(prop.key);
                });
            }
        },

        async deleteMetadata () {
            const prop = new XmpMetadataProperty();
            prop.key = 'dc:creator';
            prop.value = null;        // null value means delete property...
            prop.namespaceUri = 'http://purl.org/dc/elements/1.1/';

            const metadata = new XmpMetadata();
            metadata.properties = [prop];
            
            const response = await pdfApi.postXmpMetadata(configParams.PDF_DOCUMENT_NAME, metadata);

            if (response.body.code == 200) {

>                 console.log("Dlete metadata '" + prop.key + "' successful!");

Reviewer: ```suggestion
                console.log("Delete metadata '" + prop.key + "' successful!");
```
